doc/tutorials/using_windows_as_uos.rst

```diff
@@ -144,9 +72,9 @@ Create a raw disk
 
 Run these commands on the Service VM::
 
-   # swupd bundle-add kvm-host
-   # mkdir /root/img
-   # cd /root/img
+   # sudo apt-get install qemu
```

Comment:
I don't have my Ubuntu system at hand, can you confirm that `qemu` installs `qemu-img`? I know it's part of the `qemu-utils` package but no 100% it is installed by the `qemu` package. Thanks

Reply:
while qemu is installed, the following additional packages will be installed:
  qemu-block-extra qemu-system qemu-system-common qemu-user qemu-user-binfmt qemu-utils

Reply:
Unless you need any of those for other reasons, let's just install `qemu-utils`.